server/api/field.go

```diff
@@ -12,8 +12,9 @@ const (
 )
 
 type SelectOption struct {
-	Label string `json:"label"`
-	Value string `json:"value"`
+	Label    string `json:"label"`
+	Value    string `json:"value"`
+	IconData string `json:"icon_data"`
```

Comment:
OT: Can we get a recommendation from the (Web, Mobile)Apps team as to the best practice of handling icons - 1/5 we should adopt either embedded data, or URLs, consistently in (most) contexts.

Reply:
@levb FYI we pass the data to the React component the same way whether it's data or a URL, and it "just works". i.e.

```jsx
<img src={'data:iojriqjwiqj0iefjwe...'}/>

<img src={'https://images.com/image.png'}/>
```